ipaserver/install/server/install.py

```diff
@@ -940,7 +940,29 @@ def install(installer):
 
     print("======================================="
           "=======================================")
-    print("Setup complete")
+    print("IPA server installation successful")
+    print("")
+    print("You can access Web UI using https://%s/ipa/ui" % (host_name))
+    print("")
+    print("Obtain a Kerberos Ticket for issuing IPA command 'kinit admin'.")
+    print("")
+    print("Server is installed with following roles:")
+    print("* 389 Directory Server")
+    print("* Kerberos Server")
+    print("* HTTP Server")
+    if setup_ca:
+        print("* CA Server")
+    if options.setup_dns:
+        print("* DNS Server")
+    print("")
+    print("For troubleshooting and errors:")
+    print("Please visit : https://www.freeipa.org")
+    print("")
+    print("For documentation:")
+    print("Please visit:")
+    print("https://access.redhat.com/documentation/en-us/"
+          "red_hat_enterprise_linux/7/html/"
+          "linux_domain_identity_authentication_and_policy_guide/index")
     print("")
     print("Next steps:")
     print("\t1. You must make sure these network ports are open:")
```

Comment:
There is only one step left so either this should be dropped completely or make the CA root cert backup step 2. Of course in the case of CAless this would revert to being one step.

If you are going to point to troubleshooting should it not go to https://www.freeipa.org/page/Troubleshooting ?

This looks to be more than a screenful of data. All masters have the LDAP, KDC and HTTP "roles", is it worth displaying that?

Is there any assurance from the content people that the documentation link will never change?

Reply:
||There is only one step left so either this should be dropped completely or make the CA root cert backup step 2. Of course in the case of CAless this would revert to being one step.
I believe we can drop it.

||This looks to be more than a screenful of data. All masters have the LDAP, KDC and HTTP "roles", is it worth displaying that?
||Is there any assurance from the content people that the documentation link will never change?
Yes this need to be clarified. But Providing link to troubleshooting is good idea. Customer can/should investigate before opening an Bug. Again This page need to be updated all times with latest information. If you want to point to some other page, more upto date and maintained. I have no objections.

Reply:
Honestly I'd prefer to send them to some freeipa.org page that links to the RH docs by version. It will be a significantly shorter URL as well. Perhaps https://www.freeipa.org/page/Documentation